### PR TITLE
fix(wallet): Dont Show Aurora in Ethereum Submenu

### PR DESCRIPTION
**Pull request by Douglashdaniel**
## Description 
Fixes a bug where `Aurora` was being listed under `Ethereum` submenu

Resolves <https://github.com/brave/brave-browser/issues/26813>

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

   1. Go to the Portfolio page
   2. Click on the `Network` filter and hover over `Ethereum`
   3. `Aurora` should not be listed in the `Submenu`

Before:

https://user-images.githubusercontent.com/40611140/202241688-29d8ca1b-7473-4a6e-bdfb-37ca1d6d507d.mov

After:

https://user-images.githubusercontent.com/40611140/202241717-0652f6cc-dbf1-444d-acbc-784b848952bb.mov

### CODE REVIEW

**Review by muliswilliam — APPROVED** [review]

File: components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx [hunk 1]
```diff
@@ -122,9 +122,10 @@ export const NetworkFilterSelector = ({ networkListSubset }: Props) => {
                   {sortedNetworks.filter((n) =>
```
Comment (onyb) [inline components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx:122]: See line #120 - what happens if `isTestNetworksEnabled` is false?

File: components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx [hunk 2]
```diff
@@ -122,9 +122,10 @@ export const NetworkFilterSelector = ({ networkListSubset }: Props) => {
...
                     n.coin === network.coin &&
                     n.symbol.toLowerCase() === network.symbol.toLowerCase() &&
-                    // Optimism's native asset is considered ETH, so we want to make sure
-                    // we dont include it under Ethereum's submenu.
-                    n.chainId !== BraveWallet.OPTIMISM_MAINNET_CHAIN_ID)
+                    // Optimism and Aurora's native asset is considered ETH, so we want to make sure
```
[the remark this answers is not in the document]
Comment (Douglashdaniel) [reply components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx]: Good point! Updated to check against supported `TestNetworks`

File: components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx (hunk 1 above)
Comment (Douglashdaniel) [reply components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx]: If `isTestNetworksEnabled` is `false` we do not show the `subMenu`

**Review by onyb — APPROVED** [review]
++

**Comment by srirambv** [comment]
Verification passed on <html>
<body>


Brave | 1.47.84 Chromium: 108.0.5359.48 (Official Build) nightly (64-bit)
-- | --
Revision | `18ceeca0d99318e70c00d2e04d88aa55488b5c63-refs/branch-heads/5359@{#854}`
OS | Linux


</body>
</html>

- Verified steps from PR
- Verified Ethereum submenu only shows Ethereum testnets and doesn't list Aurora network
image

**Pull request merged**